food-truck-frontend/src/components/truck/rate_and_review/ReviewList.tsx

```diff
@@ -91,14 +86,14 @@ class ReviewListComponent extends Component<RateProps, RateState> {
   );
 
   ratingsReport = (id: number, starRating: number, costRating: number, item: boolean = false) => (
-    <Grid container item={item} xs key={`${id}/ratings`}>
+    <Grid container item={item} key={`${id}/ratings`}>
       <Grid item xs key={"star"}>
         <Typography variant="h6">Quality:</Typography>
-        <StarRating precision={0.1} disabled value={starRating} />
+        <StarRating readOnly precision={0.1} disabled value={starRating}/>
       </Grid>
       <Grid item xs key={"`cost"}>
         <Typography variant="h6">Value:</Typography>
-        <MoneyRating precision={0.1} disabled value={costRating} />
+        <MoneyRating readOnly precision={0.1} disabled value={costRating}/>
```

Comment:
I'm not sure we need both readOnly and disabled.

Reply:
readOnly is required if there is no "name" attr for a form, and gives a warning if left out